README.md

```diff
@@ -10,6 +10,31 @@ Install all dependencies
 >$ npm install
 ```
 
+Replace line 6 in `edge-frontend/profiles/local-frontend.js` to:
+
+```js
+const routes = {
+    '/config': { host: 'http://127.0.0.1:8889' },
```

Comment:
@karelhala should we put this in our local-frontend.js or the insights-proxy default routes?

Reply:
This is not mandatory, once edge is on stable branches we won't be needing it anymore. Plus if you run `BETA=true npm run start:proxy` you should be able to see the UI on `https://ci.foo.redhat.com:1337/beta/edge`